composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=5.3.0",
         "composer/installers": "*",
-        "openbuildings/phpunit-spiderling": "0.1.*",
+        "openbuildings/phpunit-spiderling": "^0.1.12",
```

Comment:
when you update `composer.json` you need to update `composer.lock` as well. You can use `composer update --lock` to update just the hash if you don't want to update the actual locked versions.